internal/host/host_test.go

```diff
@@ -2371,7 +2371,8 @@ var _ = Describe("UpdateImageStatus", func() {
 	}
 
 	tests := []struct {
-		name                  string
+		name string
```

Comment:
:confused: why change this here?

Reply:
actually it was make format that did it

Reply:
Strange that it would add a space in a test structure, but :man_shrugging: